### PR TITLE
nx-tile__sub-header should be an h3 RSC-628

### PR DESCRIPTION
**Pull request by glenhunter**
https://issues.sonatype.org/browse/RSC-628

Changes to documentation and examples. Added an `hgroup` with a class and convenience component.

### CODE REVIEW

**Comment by rpokorny** [comment]
The `nx-tile` documentation table is overflowing

**Comment by rpokorny** [comment]
Borked
image

**Comment by glenhunter** [comment]
> The `nx-tile` documentation table is overflowing

Fixed: https://github.com/sonatype/sonatype-react-shared-components/pull/457/commits/8734975abda6f46e88830c85f9befdee8112d65e

**Comment by glenhunter** [comment]
> Borked

Unborked: https://github.com/sonatype/sonatype-react-shared-components/pull/457/commits/02aff8816685f05eb10dcc251f8d24fd1c903327

**Comment by rpokorny** [comment]
Spacing around the subtitle does not match master

**Comment by glenhunter** [comment]
> Spacing around the subtitle does not match master

Crap. Removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/457/commits/bf0f15db27cef4cf0b500180333f0162fcf6c248

File: gallery/src/styles/NxTile/NxTilePage.tsx [hunk 1]
```diff
@@ -46,20 +47,36 @@ const NxTilePage = () =>
...
+            <td className="nx-cell"><code className="nx-code">NxTile.Headings</code></td>
+            <td className="nx-cell">Nested inside <code className="nx-code">.nx-tile-header</code></td>
+            <td className="nx-cell">
+              If there is a sub-title then the <NxCode>h2.nx-tile-header__title</NxCode> &amp;
+              {' '}<NxCode>h3.nx-tile-header__subtitle</NxCode> should both be wrapped in a containing
+              {' '}<code className="nx-code">&lt;hgroup&gt;</code> with this class. If there
+              is only an <code className="nx-code">.nx-tile-header__title</code> then this element and its class are
+              optional.
+            </td>
+          </tr>
```
Comment (rpokorny) [inline gallery/src/styles/NxTile/NxTilePage.tsx:61]: Should probably use the convenience components everywhere

**Review by rpokorny — APPROVED** [review]

File: gallery/src/styles/NxTile/NxTilePage.tsx (hunk 1 above)
Comment (glenhunter) [reply gallery/src/styles/NxTile/NxTilePage.tsx:61]: > Should probably use the convenience components everywhere

This page is covered in the iDay PR.

Comment (rpokorny) [reply gallery/src/styles/NxTile/NxTilePage.tsx:61]: > This page is covered in the iDay PR.

That's my point.  These new lines _aren't_ covered in that PR

Comment (glenhunter) [reply gallery/src/styles/NxTile/NxTilePage.tsx]: > > This page is covered in the iDay PR.
> 
> That's my point. These new lines _aren't_ covered in that PR

Ah yes, that could be an ugly merge conflict. Converted here: https://github.com/sonatype/sonatype-react-shared-components/pull/457/commits/2e97faac81405f7c6c05143d9047fab1a8abddab

**Pull request merged**